code/Snakefile

```diff
@@ -0,0 +1,41 @@
+# Snakefile to run Amdahl's Law for HPC Workflows
+
+cpu_min = 1
+cpu_max = 32
+
+TASKS = range(cpu_min, cpu_max + 1)  # number of processors: start small for testing!
+
+rule plot:
+    input:
+        expand("amdahl_np{task}.json", task=TASKS)
+    output:
+        "plot.log"
```

Comment:
Maybe change the extension of this output file? `.log` conveys a log file for me, i.e. one generated by the log directive.

Reply:
Ah, right, the output of the `plot` rule is the PNG file, not a text file! Updating...

Reply:
Fixed by 794f8a4